include/GeographyPoint.hpp

```diff
@@ -95,6 +95,7 @@ class GeographyPoint {
      * Return true iff this is a null GeographyPoint.
      */
     bool isNull() const {
+        static const double NULL_COORDINATE = 360.0;
```

Comment:
No, there is no other consumer of NULL_COORDINATE except this function and can not initialize double static inside header, so moved it to the function. I will move the initialization and function to the cpp file.